lens/main.scrbl

```diff
@@ -4,10 +4,10 @@
 
 @defmodule[lens]
 
-This library includes functions and forms for working with @italic{lenses}.
-A lens is a value that operates on some small piece of a larger
-structure. Think of them as a more general representation of getters and
-setters in object-oriented languages.
+This library includes functions and forms for working with
+@deftech[#:tag 'lens]{lenses}. A lens is a value that operates on some
```

Comment:
Is this supposed to be #:key "lens"? The travis build claims deftech doesn't take a #:tag keyword argument, and the docs for deftech only mention #:key